src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import duke.task.Deadlines;
+import duke.task.Events;
+import duke.task.Task;
+import duke.task.ToDos;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Storage {
+    private final File file;
+
+    public Storage(String filePath) {
+        this.file = new File(filePath);
+        file.getParentFile().mkdir();
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public List<Task> load() throws DukeException {
```

Comment:
Maybe change this to loadTasks? Since there is another method named load for the Task class